src/components/SimpleBlock.vue

```diff
@@ -0,0 +1,271 @@
+          Контент вкладки event
+        </div>
+
+        <div class="tabs__content" v-if="activeTab === 'alert'">
+          <!-- Контент вкладки alert -->
+        </div>
+      </div>
+    </div>
+
+  </div>
+</template>
+
+<script>
+export default {
+  name: 'SimpleBlock',
+  data () {
+    return {
+      tabs: {
+        standard: {
+          disabled: false
+        },
+        event: {
+          disabled: false
+        },
+        alert: {
+          disabled: true
+        }
+      },
+      activeTab: 'standard',
+      errors: [],
+      link: null
+    }
+  },
+  methods: {
+    changeTab (tab) {
+      if (!this.tabs[tab].disabled) {
+        this.activeTab = tab
+      }
+    },
+    checkForm: function (e) {
+      e.preventDefault()
+      this.errors = []
+
+      if (this.link.length >= 200) {
+        this.errors.push('Длина ссылки превышает 200 символов.')
+      }
+
+      if (this.errors.length === 0) {
+        console.log(this.link)
+      }
+    }
+  }
+}
+</script>
+
+<style scoped>
+.card {
+  width: 512px;
+  min-height: 480px;
+  margin: 0 auto;
+  padding: 36px 25px 80px 32px;
+  border-radius: 4px;
+  box-shadow: 0 8px 8px 0 rgba(119, 131, 143, 0.16);
+  background-color: #fff;
+  box-sizing: border-box;
+}
+.card__title {
+  margin: 0 2px 0 0;
+  font-size: 24px;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: -0.11px;
+  color: #000;
+}
+.card__subtitle {
+  margin: 0 0 26px 0;
+  font-size: 18px;
+  font-weight: 600;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: -0.08px;
+  color: #828e9a;
+}
+.tabs__wrapper {
+  display: grid;
+  grid-template-columns: repeat(3, 1fr);
+  column-gap: 2px;
+  border-radius: 24px;
+  overflow: hidden;
+  margin-bottom: 25px;
+}
+.tabs__label-wrapper label {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  column-gap: 8px;
+  position: relative;
+  color: #77838f;
+  background-color: #e7eaf3;
+  padding: 8px 29px 9px 30px;
+  font-size: 16px;
+  font-weight: 600;
+  line-height: 22px;
+  letter-spacing: -0.07px;
+}
+.tabs__label-wrapper label:hover {
+  opacity: 0.5;
+}
+.tabs__label-wrapper span {
+  height: 22px;
+}
+
+.tabs__label-wrapper span svg path {
+  fill: #77838f;
+}
+
+.tabs__label-wrapper input:checked + label span svg path {
+  fill: #fff;
+}
+.tabs__label-wrapper input:checked + label {
+  background-color: #377dff;
+  color: #fff;
+}
+.tabs__label-wrapper input:disabled + label {
+  opacity: 0.5;
+}
+.standard {
+  padding: 8px;
+}
+.standard__input-wrapper {
+  margin-bottom: 23px;
+}
+.standard__input-wrapper label {
+  font-size: 18px;
+  font-weight: 600;
+  line-height: 1.44;
+  letter-spacing: -0.08px;
+  color: #77838f;
+  margin: 5px 0;
+}
+.standard__input-wrapper input {
+  display: block;
+  width: 100%;
+  padding: 12px 24px;
+  box-sizing: border-box;
+  font-family: 'Nunito', Helvetica, Arial, sans-serif;
```

Comment:
you have already declared `font-family` in App.vue, you could add to App.vue also 
```
button, input {
  font-family: inherit;
}
```